src/components/App/App.tsx

```diff
@@ -1,13 +1,51 @@
-import React from 'react';
-import logo from '../src/assets/logo.svg';
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
 import './App.css';
+import mapIcon from '../../assets/WorldMap.jpg';
+import Map from '../Map/Map';
+import apiCalls from '../../api/apiCalls';
+
+interface IState{
+  countries: {
+    name: string
+    population: number
+  }[]
+}
 
 function App() {
+  const [countries, setCountries] = useState <IState["countries"]>([])
+  
+  useEffect(() => {
```

Comment:
I like using this useEffect function instead of doing it as a componentDidMount on a class component